src/data/polynomial/expand.lean

```diff
@@ -240,7 +240,7 @@ variable {R}
 
 theorem of_irreducible_expand {p : ℕ} (hp : p ≠ 0) {f : R[X]}
   (hf : irreducible (expand R p f)) : irreducible f :=
-@@of_irreducible_map _ _ _ (is_local_ring_hom_expand R hp.bot_lt) hf
+let _ := is_local_ring_hom_expand R hp.bot_lt in by exactI of_irreducible_map ↑(expand R p) hf
```

Comment:
Any idea what happened here?

Reply:
not really sure, but the typechecking has been crazy brittle with `int_cast` and stuff: for some further examples that bewilder me, look at [this](https://github.com/leanprover-community/mathlib/commit/852da3bf0f179852c0e638c324f4912ac4d67e0a) and its [fix](https://github.com/leanprover-community/mathlib/commit/29aac71a6e8c39b1550cd824703f3fff0e3ddfdd) [and no, deinstancing the `comm_semiring` instance didn't help at all]

Reply:
Either way, I guess this is nicer because it avoids an `@`